pkg/frontend/transport/handler.go

```diff
@@ -200,6 +201,9 @@ func (f *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			writeError(w, apierror.New(apierror.TypeInternal, err.Error()))
 			return
 		}
+		ctx, _ := context.WithDeadlineCause(r.Context(), deadline,
+			cancellation.NewErrorf("write deadline exceeded (timeout: %v)", f.cfg.ActiveSeriesWriteTimeout))
```

Comment:
If ActiveSeriesWriteTimeout is set to `0` (meaning to use `server.http-write-timeout`), this will log incorrectly.

Reply:
This whole block is guarded by `f.cfg.ActiveSeriesWriteTimeout > 0` though, so this code will not execute if the flag is set to `0`, or am I missing something? 🤔

Reply:
Ah, yes, I was looking at the previous change and didn't pay enough attention here. Thanks!